composer/trainer/trainer.py

```diff
@@ -491,6 +491,25 @@ class Trainer:
             Ignored if ``load_path`` is either ``None`` or a local file path. (default: ``1,048,675``)
         load_progress_bar (bool, optional): Display the progress bar for downloading the checkpoint.
             Ignored if ``load_path`` is either ``None`` or a local file path. (default: ``True``)
+        load_ignore_keys (List[List[str]] | (Dict) -> None, optional): A list of paths for the ``state_dict``,
+            which, when provided, will be ignored from the state_dict before a checkpoint is loaded. Each path is a list
+            of strings specifying the keys to index into ``state_dict``. If a prefix is provided, all children are also
+            ignored.
+
+            Example 1: `load_ignore_model_keys = [["state", "model", "classifier", "weights"], "state", "model", "classifier", "bias"]]`
```

Comment:
Hmm, this nested dictionary isn't actually super clear. I'm fine with it, but I don't think that it is the best possible UX. Could we do a string that separates on some character instead, or wildcard syntax?

Reply:
lol I think I pushed the change to `/` separator after you started the review but before you finished it -- tough thread race :'(